krun/platform.py

```diff
@@ -963,9 +977,12 @@ def _check_isolcpus(self):
 
             break  # found the isolcpus arg
         else:
-            self._advise_isolcpus_arg([], expect_cpus)  # exits
+            v = ""  # didn't find a isolcpus arg
 
-        got_cpus = list(sorted(v.split(",")))
+        if v == "":
+            got_cpus = []
+        else:
+            got_cpus = list(sorted(v.split(",")))
```

Comment:
I would be inclined to move `got_cpus = ` into the previous blocks, not just to avoid the extra if/else but by the time I got to here, I have forgotten exactly what `v` should look like. YMMV.